### PR TITLE
Home task 2. 

### PR DESCRIPTION
**Pull request by JackVeromeev**
I'm very sorry for being late and ready to buy some bear.

### CODE REVIEW

**Comment by JackVeromeev** [comment]
I'm sorry. I was staring at the same error. Fortunately, all errors were fixed

File: 2015/YakovVeromeev/Homework-2/source/filter.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Filter
+  def self.filter(versions, parametr)
```
Comment (JackVeromeev) [inline 2015/YakovVeromeev/Homework-2/source/filter.rb]: There's only 2 parametrs the method accepts. Is it too much?

File: 2015/YakovVeromeev/Homework-2/bin/gemfiler [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+Dir["../source/*.rb"].each do |file| # incluse all source code files form ../source/
+  require_relative file
+end
+
+unless ARGV.size == 2
+  raise Argument_Number_Error # check if application accepted wrong number of arguments
```
Comment (Saicheg) [inline 2015/YakovVeromeev/Homework-2/bin/gemfiler]: такой ошибки нету

File: 2015/YakovVeromeev/Homework-2/bin/gemfiler [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+end
+
+name, param = ARGV
```
Comment (Saicheg) [inline 2015/YakovVeromeev/Homework-2/bin/gemfiler]: нет работы с библиотекой, как я просил

**Pull request closed without merging**